tests/cam/cam.cpp

```diff
@@ -46,6 +45,7 @@ main(int argc, char *argv[])
 	}
 
 	for (;;) {
+		size_t i = 0;
```

Comment:
Still no. This should be declared *inside* the `for` declaration if it is not used outside the loop.

``` c++
 		for (size_t i = 0; i < circles.size(); i++) {
 			Vec3i c = circles[i];
 			circle(fm, Point(c[0], c[1]), 2, Scalar(0, 255, 0),
 				3, LINE_AA);
                }
```

Moreover, can't you directly iterate over `circles` rather than using an index? Something like

``` c++
   for (auto const& c: circles) {
      …
   }
```